sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcSourceSuite.scala

```diff
@@ -96,8 +97,10 @@ class HiveOrcSourceSuite extends OrcSuite with TestHiveSingleton {
            |STORED AS orc
            |LOCATION '$uri'""".stripMargin)
       val result = Row("a", "b         ", "c", Seq("d  "))
-      checkAnswer(spark.table("hive_orc"), result)
-      checkAnswer(spark.table("spark_orc"), result)
+      withSQLConf(HiveUtils.CONVERT_METASTORE_ORC.key -> "false") {
```

Comment:
Sorry, @gatorsmile . I missed that this test case assumes that `convertMetastoreOrc=false`.
It's my bad. Actually, I had a patch to support `CHAR` in `convertMetastoreOrc` internally, so I was confused.

I'm investigating the root cause why we missed this. There seems to be some interference between test suites about SQLConf. So far, I found the test passed strangely like the following.
```
build/sbt -Phive "project hive" "test-only *.HiveOrcFilterSuite *.HiveOrcSourceSuite"
```
But, this fails when we do this alone.
```
build/sbt -Phive "project hive" "test-only *.HiveOrcSourceSuite"
```

First of all, I'll try to find the root case among test suites. Then, I'll make the ORC CHAR patch later.